ArcMovies.xcodeproj/xcuserdata/Yuri.xcuserdatad/xcdebugger/Breakpoints_v2.xcbkptlist

```diff
@@ -0,0 +1,5 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<Bucket
+   type = "1"
```

Comment:
Why you commit this file?

Reply:
That was actually my bad, I didn't add a .gitignore to the project, this file wasn't supposed to be committed.